docs/collecting-data/collecting-from-own-applications/react-native-tracker/hybrid-apps/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Hybrid Apps"
+date: "2022-08-30"
+sidebar_position: 70
+---
+
+# Hybrid Apps
+
+:::info
+
+This feature is available since v1.3.
+
+:::
+
+Hybrid apps are mobile apps that in addition to a React Native interface, provide part of the UI through an embedded Web view. Snowplow events are tracked from both the React Native code as well as the Web view. Our goal is to have both events tracked from both places to share the same session and appear as tracked with the same tracker.
+
+This use case is supported by implementing the [Snowplow WebView tracker](../../webview-tracker/index.md) in the Web view in your app. The WebView tracker is able to pass events to the React Native tracker which sends them to the collector.
+
+Please refer to the [Snowplow Hybrid Apps Tracking tutorial](https://snowplow-incubator.github.io/snowplow-hybrid-apps-accelerator/) for a step-by-step guide how to set up tracking in hybrid apps.
```

Comment:
We can give it a nicer link:
* either add a 302 redirect in `static/_redirects` from docs.snowplow.io/accelerators/hybrid-apps to this address,
* or we deploy this accelerator with Netlify like we did with the other one, and add a 200 redirect (which behaves like a reverse proxy)

@paulboocock could you help @matus-tomlein with this?

Reply:
This accelerator should be on https://docs.snowplow.io/accelerators/hybrid/ once the other PRs in the acclerator and this repo (#109) go live.